### ISSUE TITLE
AvoidEscapedUnicodeCharacters should support '\s' escape symbol

### ISSUE THREAD

**Issue by romani** [checkstyle/checkstyle#9133]
https://checkstyle.org/config_misc.html#AvoidEscapedUnicodeCharacters

blocked by https://github.com/checkstyle/checkstyle/issues/9132

```
$ cat config.xml 
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">

<module name="Checker">
    <property name="charset" value="UTF-8"/>

    <module name="TreeWalker">
      <module name="AvoidEscapedUnicodeCharacters">
        <property name="allowIfAllCharactersEscaped" value="true"/>
      </module>

    </module>
</module>

$ cat Test.java 
public class Test {
  String value = "\u03bc\t"; // ok
  String value = "\u03bc\s"; // ok
}

```

Expected: no violations.

**Comment by nmancus1** [comment]
I'm on this.

**Comment by Vyom-Yadav** [comment]
Also if I use the configuration-
```
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
        "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
        "http:///www.puppycrawl.com/dtds/configuration_1_3.dtd">
<module name="Checker">
    <module name="TreeWalker">
        <module name="AvoidEscapedUnicodeCharacters">
            <property name="allowNonPrintableEscapes" value="FALSE"/>
        </module>
        </module>
    </module>

```
With code-
```
public class Test {
    public static void main(String[] args) {
        String unitAbbrev = "\t";

    }

}
```
It Shows Audit Done with 0 errors. Property is defined as allow use escapes for non-printable, whitespace characters when its `value="FALSE"` it shouldn't allow the use of escapes for non-printable, whitespace characters. But no error is shown.

**Comment by rnveach** [comment]
Fix was merged
